pkg/common/utils/resource/pod.go

```diff
@@ -729,6 +752,46 @@ func getMultiSecretVolumeAndVolumeMount(bSpec *v1.BaseSpec, componentType v1.Com
 	return volumes, volumeMounts
 }
 
+func GetMultiSecretVolumeAndVolumeMountWithCommonSpec(cSpec *dv1.CommonSpec, componentType dv1.DisaggregatedComponentType) ([]corev1.Volume, []corev1.VolumeMount) {
```

Comment:
componentType is not a must parameter

Reply:
if only log, the componentType can omit int parameters. log in message on the call function.

Reply:
It's better to make judgment and print log in the `GetMultiSecretVolumeAndVolumeMountWithCommonSpec` here. Because there are three statefulsets in the disaggregated cluster, and each statefulset has two methods that call the `GetMultiSecretVolumeAndVolumeMountWithCommonSpec`, which results in numerous places that need to be modified.

Reply:
It's not needed to output a log.  common spec only in fe, be, cn, broker spec, so, if the common spec is not empty, the component type is exist.

Reply:
Done, `componentType` has been removed.